website/docs/r/identity_group.html.md

```diff
@@ -60,6 +60,8 @@ The following arguments are supported:
 
 * `external_policies` - (Optional) `false` by default. If set to `true`, this resource will ignore any policies return from Vault or specified in the resource. You can use [`vault_identity_group_policies`](identity_group_policies.html) to manage policies for this group in a decoupled manner.
 
+* `external_member_entity_ids` - (Optional) `false` by default. If set to `true`, this resource will ignore any policies return from Vault or specified in the resource. You can use [`vault_identity_group_member_entity_ids`](identity_group_member_entity_ids.html) to manage member entities for this group in a decoupled manner.
```

Comment:
Suggest to replace: 
1. `will ignore any policies return` with `will ignore any Entity IDs returned`
2. `to manage member entities for` with `to manage Entity IDs for`

Reply:
Fixed